src/firefly/js/visualize/ui/MouseReadout.jsx

```diff
@@ -470,12 +255,15 @@ function getSingleMouseReadout(plot, imagePt, toCoordinateName) {
                 break;
 
             case CoordinateSys.PIXEL:
-                var pt = plot.projection.getPixelScaleArcSec();
-                result = `  ${numeral(pt).format(precision3Digit)}"`;
+                obj =readoutItems.pixel;
+                fStr = '0.' + new Array(obj.precision + 1).join('0');
+                result = `${numeral(obj.value).format(fStr)} ${obj.unit || ''}`;
+
```

Comment:
Great suggestion!!! I like it.


On 6/4/16 7:22 AM, Trey Roby wrote:
> |const myFormat= (v,precision) => 
> numeral(v).format(padEnd('0.',precision+2,'0') );|

-- 
Lijun Zhang

LSST/IPAC
California Institute of Technology
770 S. Wilson Ave, Mail Code 100-22
Pasadena, CA 91125, USA
Voice: 1-626-395-1972 Fax:1-626-395-1816